src/components/activity-page/activity-page-content.tsx

```diff
@@ -57,14 +57,15 @@ export class ActivityPageContent extends React.PureComponent <IProps, IState> {
     const secondaryIsOnLeft = page.layout === PageLayouts.Responsive || page.layout === PageLayouts.SixtyForty;
     const collapsible = page.toggle_info_assessment && page.layout !== PageLayouts.FullWidth;
     const renderSecondary = this.renderSecondaryEmbeddables(visibleEmbeddables.infoAssessment, questionsBeforeSecondary, page.layout, secondaryIsOnLeft, collapsible);
+    const pageTitle = page.name || ``;
 
     const [first, second] = primaryFirst
                             ? [renderPrimary, renderSecondary]
                             : [renderSecondary, renderPrimary];
 
     return (
       <div className={`page-content ${page.layout === PageLayouts.Responsive ? "full" : ""}`} data-cy="page-content">
-        <div className="name">{page.name}</div>
+        <div className="name">{`Page ` + pageNumber + `: ` + pageTitle}</div>
```

Comment:
I wonder if the completion page should have a page number?  It doesn't at present.  Not sure if it should or not.